uxarray/grid/connectivity.py

```diff
@@ -230,3 +232,104 @@ def _build_face_edges_connectivity(grid):
             "start_index": INT_DTYPE(0),
             "long_name": "Maps every edge to the two nodes that it connects",
         })
+
+
+def _build_node_faces_connectivity(grid):
```

Comment:
Should this be added to the internal_api?

Reply:
I added it to the internal_api.rst under the `helper` section before the redesign. However, I am open to @philipc2 opinion about where this function should belong to.

Reply:
Good catch @aaronzedwick, that function should be listed under here:

https://github.com/UXARRAY/uxarray/blob/a08e11c2dc7aefb8371c49cb381689740792983e/docs/internal_api/index.rst?plain=1#L114-L126C26